src/appUtils.spec.ts

```diff
@@ -0,0 +1,655 @@
+import * as fs from "fs-extra";
+import { expect } from "chai";
+import {
+  extractAppIdentifiersFlutter,
+  extractAppIdentifierIos,
+  extractAppIdentifiersAndroid,
+  Platform,
+  getPlatformsFromFolder,
+  detectApps,
+  App,
+} from "./appUtils";
+
+const FLUTTER_CONFIG = `
```

Comment:
Rather than having these configs as constants and setting up new directories every time, we should just add the test directories to the src/test/fixtures directory

Reply:
Spoke about this offline - recording the decision here: 
There are a few too many permutations to make using test fixtures reasonable, but I have switched this to using mockfs to limit file creation.